packages/tasks/src/model-libraries.ts

```diff
@@ -422,6 +422,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
 		snippets: snippets.tensorflowtts,
 	},
+	"tic-clip": {
+		prettyLabel: "tic-clip",
+		repoName: "tic-clip",
+		repoUrl: "https://github.com/apple/ml-tic-clip",
+		filter: false,
+		countDownloads: { term: { path: "checkpoints/*.pt" } },
```

Comment:
This should be a wildcard as  https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts#L111

```suggestion
		countDownloads: { wildcard: { path: "checkpoints/*.pt" } },
```